HLTriggerOffline/Higgs/python/hltHiggsPostProcessors_cff.py

```diff
@@ -320,6 +320,29 @@ def get_reco_strings(strings):
 hltHiggsPostWHToENuBB.subDirs = ['HLT/Higgs/WHToENuBB']
 hltHiggsPostWHToENuBB.efficiencyProfile = efficiency_strings_WHToENuBB
 
+#Specific plots for MSSMHbb
+#Jet plots
+NminOneCuts = (_config.__getattribute__("MSSMHbb")).__getattribute__("NminOneCuts")
```

Comment:
would not like to use this backdoor for the config - why can the the pset not just be imported?